rulesets/faang_experiments.metadata_rules.json

```diff
@@ -7,7 +7,7 @@
       "name": "standard",
       "rules": [
         {
-          "mandatory": "mandatory",
+          "mandatory": "recommended",
```

Comment:
Surely this should be mandatory?

Reply:
as discussed, it is impossible to make it mandatory at the moment